apache/client/include/lauth/json.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef __LAUTH_JSON_HPP__
+#define __LAUTH_JSON_HPP__
+
+#undef JSON_DIAGNOSTICS
+#define JSON_DIAGNOSTICS 1
+#include <nlohmann/json.hpp>
+using json = nlohmann::json;
```

Comment:
If we are going to do the `using` alias in this header, it should go into our namespace. Operating within a `namespace mlibrary::lauth` block elsewhere will allow the same usage (unscoped reference), while not adding this symbol to the global scope.